Webex Connect Flows/v3.3/Template/Apple Business Messages/README.MD

```diff
@@ -0,0 +1,81 @@
+- Select "Receive Apple Messages for Business" as the event
+- Configure the "From" field as ```$(<nodeId>.abc.abcUserId)```
+- Select the "Event name" as "Interactivemessageresponse: FormMessage"
+
+#### Step 3: Adding the form response to the conversation history using the Append Conversation node
+
+- In the Append Conversation node, select "Single Message"
+- Select "Message Type" as "AMB Form Response"
+- Set "Form response" as ???
+- Set "Timestamp" as ```$(<nodeId>.abc.timestamp)```
+
+#### Step 4: Extracting the values in the form via the Evaluate Node
+
+![AppleFormEvalNode.png](../../images/AppleFormEvalNode.png)
+- In this sample flow, the 3 fields presented in the form are being extracted in the Evaluate node with the below script. Similar script can be developed to extract any other fields configured by the flow admin
+- Update if required ???
+- Add the below lines to the beginning of the script:
+    ```
+    if (formResponse !== null && formResponse !== "") {
+        var responseJson = JSON.parse(formResponse);
+
+        for (i = 0; i < responseJson.length; i++) {
+            if (responseJson[i].pageIdentifier == "name" && responseJson[i].items[0].value !== "" && responseJson[i].items[0].value !== null)
+                var customerName = responseJson[i].items[0].value;
+            else if (responseJson[i].pageIdentifier == "emailAddress" && responseJson[i].items[0].value !== "" && responseJson[i].items[0].value !== null)
+                var customerEmail = responseJson[i].items[0].value;
+            else if (responseJson[i].pageIdentifier == "gender")
+                var gender = responseJson[i].items[0].value;
+        }
+    }
+    ```
+
+#### Further steps
+
+![AppleFormFlowOverview.png](../../images/AppleFormFlowOverview.png)
+- Rest of the flow continues as usual, Resolving the conversation, Queuing the task and so on
+- Optionally, the flow admin can choose to set variables for the fields extracted from the form and make them visible to / editable by the agent, using the Set Variable node. For further details on how to use this node, refer here - Webex Connect Flows/Sample/Usage of Set Variable In Flows/README.md
+- Note: If the customer name is made available through the form, the contact origin will be set as the name itself. This will be visible on the Agent's desktop and in reports as well. If the customer's name is not available, the userId will be used as the origin.
+
+
+
+
+## Overview of Apple unsubscribe flow
+
+- This workflow will be triggered whenever a customer unsubscribes from the business's messages.
+- This flow takes care of closing the task associated with the conversation.
+- Once the customer has chosen to unsubscribe, the business cannot send messages to the customer via the agent or the flow until he messages the business.
+- Hence, this flow can be modified to perform any post contact treatments if needed (API calls, DB dips etc.), but any attempt to contact the customer will fail.
```

Comment:
This flow will not be used for post contact treatment, that will still be the task closed flow for surveys etc. I guess we dont need to mention this point, we can just say that the flow can be imported as is to support unsubscribe.